composer.json

```
@@ -0,0 +1,65 @@
+{
+    "name": "wordpress/php-ai-client",
+    "description": "A provider-agnostic PHP SDK for communicating with generative AI models",
+    "license": "GPL-2.0-or-later",
+    "type": "library",
+    "keywords": [
+        "ai",
+        "api",
+        "llm"
+    ],
+    "authors": [
+        {
+            "name": "WordPress AI Team",
+            "homepage": "https://make.wordpress.org/ai/"
+        }
+    ],
+    "homepage": "https://github.com/WordPress/php-ai-client",
+    "support": {
+        "issues": "https://github.com/WordPress/php-ai-client/issues",
+        "source": "https://github.com/WordPress/php-ai-client"
+    },
+    "require": {
+        "php": ">=7.4 || ^8.0",
+        "ext-json": "*"
+    },
+    "require-dev": {
+        "dealerdirect/phpcodesniffer-composer-installer": "^1.0",
+        "phpstan/phpstan": "^1.10",
+        "phpunit/phpunit": "^9.5 || ^10.0",
+        "squizlabs/php_codesniffer": "^3.7"
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "autoload": {
+        "psr-4": {
+            "WordPress\\AI\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "WordPress\\AI\\Tests\\": "tests/"
+        }
+    },
```

The namespace will need to be something more concrete than just "AI", as this is only the AI _client_ SDK, not a general all-in-one AI SDK. I think `AiClient` makes most sense.
```suggestion
    "autoload": {
        "psr-4": {
            "WordPress\\AiClient\\": "src/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "WordPress\\AiClient\\Tests\\": "tests/"
        }
    },
```